src/protocols/rdp/gdi.c

```diff
@@ -190,6 +193,45 @@ BOOL guac_rdp_gdi_desktop_resize(rdpContext* context) {
 
     guac_display_layer_close_raw(default_layer, current_context);
 
+    /* Make json string containing monitor information */
+    char json[JSON_BUFFER_SIZE];
+    int pos = 0;
+    pos += snprintf(json + pos, JSON_BUFFER_SIZE - pos, "{");
```

Comment:
nit: While there may be concerns about grabbing an external dependency, when it comes to "never re-invent the wheel" any manual string manipulation to do json generation/parsing is king.  Libraries like

https://github.com/nlohmann/json

will allow "C++ like" json parsing and generation that covers all the corner cases that JSON hides from the average user.